screen.go

```diff
@@ -32,43 +32,32 @@ type clearScreenMsg struct{}
 // model's Init function. To initialize your program with the altscreen enabled
 // use the WithAltScreen ProgramOption instead.
 func EnterAltScreen() Msg {
-	return enterAltScreenMsg{}
+	return enableMode(ansi.AltScreenBufferMode)
 }
 
-// enterAltScreenMsg in an internal message signals that the program should
-// enter alternate screen buffer. You can send a enterAltScreenMsg with
-// EnterAltScreen.
-type enterAltScreenMsg struct{}
-
 // ExitAltScreen is a special command that tells the Bubble Tea program to exit
 // the alternate screen buffer. This command should be used to exit the
 // alternate screen buffer while the program is running.
 //
 // Note that the alternate screen buffer will be automatically exited when the
 // program quits.
 func ExitAltScreen() Msg {
-	return exitAltScreenMsg{}
+	return disableMode(ansi.AltScreenBufferMode)
 }
 
-// exitAltScreenMsg in an internal message signals that the program should exit
-// alternate screen buffer. You can send a exitAltScreenMsg with ExitAltScreen.
-type exitAltScreenMsg struct{}
-
 // EnableMouseCellMotion is a special command that enables mouse click,
 // release, and wheel events. Mouse movement events are also captured if
 // a mouse button is pressed (i.e., drag events).
 //
 // Because commands run asynchronously, this command should not be used in your
 // model's Init function. Use the WithMouseCellMotion ProgramOption instead.
 func EnableMouseCellMotion() Msg {
-	return enableMouseCellMotionMsg{}
+	return sequenceMsg{
```

Comment:
We might want to rename this to something like `ansiSequenceMsg` internally so it doesn't get confused with `Sequence`, the `Cmd`.

Reply:
`sequenceMsg` is the same message type we use with `tea.Sequence`. The idea here is to write the mouse sequences in order.